backend/app/api/endpoints/knowledge.py

```
@@ -767,3 +772,124 @@ def get_qa_history(
+    )
+
+    return DocumentSummaryResponse(
+        document_id=document_id,
+        summary=summary,
+    )
+
+
+@document_router.post("/{document_id}/summary/refresh", response_model=dict)
+def refresh_document_summary(
+    document_id: int,
+    data: DocumentSummaryRefreshRequest = DocumentSummaryRefreshRequest(),
```

Ruff (0.14.10)

Do not perform function call `DocumentSummaryRefreshRequest` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)